### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by WozzyaQ**
Cover `FileUtils` with tests (80% code coverage IDEA metric) and the ability to execute tests with command `sbt test`.

### CODE REVIEW

File: Jenkinsfile-build [hunk 1]
```diff
@@ -21,15 +23,17 @@ pipeline {
...
-                echo 'Testing..'
+                sh 'sbt test'
             }
         }
+
         stage('Store') {
             steps {
-                 withAWS(region: 'us-east-2',credentials: 'sigma-aws-creds') {
-                    script{
+                withAWS(region: 'us-east-2', credentials: 'sigma-aws-creds') {
```
Comment (GreenBorsch) [inline Jenkinsfile-build]: region should be param

File: Jenkinsfile-build [hunk 2]
```diff
@@ -21,15 +23,17 @@ pipeline {
...
+                    script {
                         def sourceCrawlerJar = "./jars/crawler.jar"
```
Comment (GreenBorsch) [inline Jenkinsfile-build]: use jenkins env vars instead of ./ please

File: Jenkinsfile-deploy [hunk 3]
```diff
@@ -1,6 +1,9 @@
+#!/usr/bin/env groovy
+
 pipeline {
     agent any
 
+     //TODO latest build_id (latest directory s3)
```
Comment (GreenBorsch) [inline Jenkinsfile-deploy]: TODO)

File: crawler/src/test/java/org/ua/wozzya/CrawlerTest.java [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        }
+
+
+        String key = storePrefix + "/" + DateUtils.getDateStringNowOfPattern(DateUtils.yyyyMMdd) + "/";
+
+        String actualString = S3Utils.getObjectLines(s3, storeBucketName, key + "sample")
+                .stream().collect(Collectors.joining(System.lineSeparator()));
+
+        assertTrue(s3.doesObjectExist(storeBucketName, key + "_SUCCESS"));
+        assertEquals(EXPECTED_STRING, actualString);
```
Comment (GreenBorsch) [inline crawler/src/test/java/org/ua/wozzya/CrawlerTest.java]: namings

File: Jenkinsfile-build (hunk 1 above)
Comment (WozzyaQ) [reply Jenkinsfile-build]: got it.

File: Jenkinsfile-deploy (hunk 3 above)
Comment (WozzyaQ) [reply Jenkinsfile-deploy]: done :)

**Pull request merged**